### PR TITLE
App VueJS 3 with typescript

### PR DESCRIPTION
**Pull request by shairyar**
This PR adds a test app that is using the latest version of vuejs with typescript

### CODE REVIEW

File: javascript/vue-3-ts/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# TODO
+
+Please describe what this test setup replicates.
```
Comment (luismiramirez) [inline javascript/vue-3-ts/README.md:3]: A brief description here would be nice

**Comment by luismiramirez** [comment]
The test setups must include a couple of routes to test them on CI.

The routes are:

- `/`: A root route that works
- `/error`: Throws an error when requested
- `/slow`: It takes more than 2 seconds te respond when requested

The [semaphore file](https://github.com/appsignal/test-setups/blob/main/.semaphore/semaphore.yml) has to be updated as well so the CI knows.

**Comment by unflxw** [comment]
@luismiramirez The front-end ones only need `/` to pass the tests ([see this react run as an example](https://appsignal.semaphoreci.com/jobs/4f0cefef-6242-4eaf-ab93-86f637675299#L1034))

In practice, all we're testing for the front-end setups is that they boot up a server, which implies they at least compile successfully. This is because, right now, the testing is done with an HTTP client, not a browser, so the front-end code isn't executed.

I do agree that it would be very nice if there was a way to trigger an error from the application, and if the framework has a routing system, `/error` would be a good name for it.

**Comment by shairyar** [comment]
I agree with having a separate path for `error` and `slow` but the case here is a little different, when I launch the app it does not launch because of the error. So if you are just testing if the app boots up or not then this is the correct setup to test it.

Once the issue with bootup is solved we can then have those routes.

**Comment by luismiramirez** [comment]
@shairyar ah that's true, then I think we should wait until the issue is fixed to spend more work on this setup

**Comment by tombruijn** [comment]
We have merged a test app here: https://github.com/appsignal/frontend-test-setups/tree/main/frameworks/vue/3
Closing this PR

**Pull request closed without merging**